examples/wasm/BUILD

```diff
@@ -8,10 +8,31 @@ licenses(["notice"])  # Apache 2
 
 envoy_package()
 
+filegroup(
+    name = "configs",
+    srcs = glob(
+        [
+            "**/*.yaml",
+            "**/*.wasm",
```

Comment:
this only works on amd64, im wondering how i can exclude it from other arch builds

Reply:
this should be fixed